app/javascript/components/applications/application_form/preferences/utils.js

```diff
@@ -10,7 +10,7 @@ export const naturalKeyFromMember = (member) => {
 }
 
 export const memberNameFromPref = (p) => {
-  if (p && p.application_member) {
+  if (!isEmpty(p) && p.application_member) {
```

Comment:
@akegan is there a reason we don't do a `!isEmpty` check on `p.application_member` here?